Modules/StudyProgramme/classes/model/Settings/class.ilStudyProgrammeSettingsDBRepository.php

```diff
@@ -306,14 +305,16 @@ protected function createByRow(array $row) : ilStudyProgrammeSettings
             new \ilStudyProgrammeValidityOfAchievedQualificationSettings(null, null, null)
         ;
         $automail = new \ilStudyProgrammeAutoMailSettings(false, null, null);
+        $acces_ctrl_by_orgu_positions = new \ilStudyProgrammeAccessCtrlByOrguPositionSettings();
```

Comment:
```suggestion
        $access_ctrl_by_orgu_positions = new \ilStudyProgrammeAccessCtrlByOrguPositionSettings();
```

And in other locations as well.

Reply:
Sry, i thought i clean that up already.
Done